modules/ebi-metagenomics/hhsuite/buildhhdb/tests/main.nf.test.snap

```diff
@@ -0,0 +1,85 @@
+{
+    "test_a3m_cs219.ffindex": {
+        "content": [
+            "test_a3m_cs219.ffindex:md5,1893a6d1cf11667bbb65520f5e8d6497"
+        ],
+        "timestamp": "2024-06-10T13:57:44.959353034"
+    },
+    "test_a3m_cs219.ffdata": {
+        "content": [
+            "test_a3m_cs219.ffdata:md5,0a03ab702bc958b26fba34f8d1c5fce9"
+        ],
+        "timestamp": "2024-06-10T13:57:44.956413336"
+    },
+    "test_a3m_a3m.ffdata": {
+        "content": [
+            "test_a3m_a3m.ffdata:md5,1d3e01ef4418ae118cce3e43cd5669aa"
+        ],
+        "timestamp": "2024-06-10T13:57:44.947206544"
+    },
+    "a3m - stub": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": false
+                        },
+                        [
+                            "test_a3m_a3m.ffdata:md5,d41d8cd98f00b204e9800998ecf8427e",
+                            "test_a3m_a3m.ffindex:md5,d41d8cd98f00b204e9800998ecf8427e",
+                            "test_a3m_cs219.ffdata:md5,d41d8cd98f00b204e9800998ecf8427e",
+                            "test_a3m_cs219.ffindex:md5,d41d8cd98f00b204e9800998ecf8427e",
+                            "test_a3m_cs219.log1:md5,d41d8cd98f00b204e9800998ecf8427e",
+                            "test_a3m_hmm.ffdata:md5,d41d8cd98f00b204e9800998ecf8427e",
+                            "test_a3m_hmm.ffindex:md5,d41d8cd98f00b204e9800998ecf8427e"
```

Comment:
I'm pretty sure the md5sum `d41d8cd98f00b204e9800998ecf8427e` is an empty file, is this expected?

Reply:
Yeye, this is just the stub part of the snapshot!